test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -401,6 +402,11 @@ def get_pbs_conf_file(self, hostname=None):
             if ((ret['rc'] != 0) and (len(ret['out']) > 0) and
                     (ret['out'][0] != 'False')):
                 dflt_conf = ret['out'][0]
+            """
+            print("******Executing else from get_conf_file****")
```

Comment:
@Klovely I know this is PR WIP, but I don't think we need to fix anything in this method as I don't see any problem with current code (unless I am missing something), but problem with fix. We need to get PBS_CONF_FILE value on given host (aka hostname) and it can be remote host also. But you current fix will always get PBS_CONF_FILE value from current host (where you are invoking pbs_benchpress).

Try your fix on this setup:
2 hosts, 1st host - server/sched/comm install, 2nd host mom install on non-default location and change rename /etc/pbs.conf to /etc/pbs_xyz.conf, and make sure if you do `ssh host2 echo \$PBS_CONF_FILE` it returns /etc/pbs_xyz.conf (you can use /etc/profile.d or ~/.bashrc to setup this)

Now invoke pbs_benchpress on host1 as `pbs_benchpress -t SmokeTest -p "servers=host1;nomom=host1;moms=host2"`